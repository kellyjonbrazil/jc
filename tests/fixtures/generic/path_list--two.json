[
  {
    "path": "/abc/def/gh.txt",
    "parent": "/abc/def",
    "filename": "gh.txt",
    "stem": "gh",
    "extension": "txt",
    "path_list": [
      "/",
      "abc",
      "def",
      "gh.txt"
    ]
  },
  {
    "path": "/xyz/uvw/ab.app",
    "parent": "/xyz/uvw",
    "filename": "ab.app",
    "stem": "ab",
    "extension": "app",
    "path_list": [
      "/",
      "xyz",
      "uvw",
      "ab.app"
    ]
  }
]