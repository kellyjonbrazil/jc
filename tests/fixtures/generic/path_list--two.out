/abc/def/gh.txt:/xyz/uvw/ab.app